### PR TITLE
Improved Markdown Jernej-Kavka.md

### PR DESCRIPTION
**Pull request by tiagov8**

### CODE REVIEW

File: Jernej-Kavka/Jernej-Kavka.md [hunk 1]
```diff
@@ -7,71 +7,67 @@ quote: "Once upon a time, you had to be rich and powerful to change the world. N
...
-| ![docker](../badges/Developer-docker.png)
+| ![Docker](../badges/Developer-docker.png)
 
 [[imgBadge]]
-| ![dotnet_foundation_logo.png](../badges/Event-global-ai-community.jpg)
+| ![](../badges/Event-global-ai-community.jpg)
 
 [[imgBadge]]
-| ![dotnet_foundation_logo.png](../badges/Developer-dotnet-foundation.png)
+| ![](../badges/Developer-dotnet-foundation.png)
```
Comment (jernejk) [inline Jernej-Kavka/Jernej-Kavka.md:50]:
```suggestion
| ![.NET Foundation](../badges/Developer-dotnet-foundation.png)
```

File: Jernej-Kavka/Jernej-Kavka.md [hunk 2]
```diff
@@ -7,71 +7,67 @@ quote: "Once upon a time, you had to be rich and powerful to change the world. N
...
 [[imgBadge]]
 | ![dotnetcode](../badges/Developer-dotnet-code.png)
 
 [[imgBadge]]
-| ![docker](../badges/Developer-docker.png)
+| ![Docker](../badges/Developer-docker.png)
 
 [[imgBadge]]
-| ![dotnet_foundation_logo.png](../badges/Event-global-ai-community.jpg)
+| ![](../badges/Event-global-ai-community.jpg)
```
Comment (jernejk) [inline Jernej-Kavka/Jernej-Kavka.md:47]:
```suggestion
| ![Global AI Community](../badges/Event-global-ai-community.jpg)
```

File: Jernej-Kavka/Jernej-Kavka.md [hunk 3]
```diff
@@ -7,71 +7,67 @@ quote: "Once upon a time, you had to be rich and powerful to change the world. N
...
-| ![angular.png](../badges/Developer-blazor.png)
+| ![Blazor](../badges/Developer-blazor.png)
 
 [[imgBadge]]
-| ![angular.png](../badges/Developer-angular.png)
+| ![Angular](../badges/Developer-angular.png)
 
 [[imgBadge]]
-| ![ML.NET-logo.png](../badges/Developer-dotnet-machine-learning.png)
+| ![.NET](../badges/Developer-dotnet-machine-learning.png)
```
Comment (jernejk) [inline Jernej-Kavka/Jernej-Kavka.md:29]:
```suggestion
| ![ML.NET](../badges/Developer-dotnet-machine-learning.png)
```

**Review by jernejk — CHANGES_REQUESTED** [review]
Just needs these changes, so that alt text is correct.

**Review by bradystroud — APPROVED** [review]

**Review by jernejk — APPROVED** [review]
LGTM

**Pull request merged**